prime-angular-frontend/src/app/modules/health-auth-site-reg/health-auth-site-reg.routes.ts

```diff
@@ -103,12 +103,12 @@ export class HealthAuthSiteRegRoutes {
    */
   public static siteSteps(): object[] {
```

Comment:
Rather than `object` can a type be defined?

Reply:
Good catch! it should be fixed